admin_guide/backup_restore.adoc

```diff
@@ -255,10 +255,12 @@ and displays {product-title}'s configuration:
 [[cluster-restore-multiple-member-etcd-clusters]]
 == Cluster Restore for Multiple-member etcd Clusters
 
-When using an external etcd host, you must first restore the etcd backup
-by creating a new, single node etcd cluster. If using external etcd with
-multiple members, you must then also add any additional etcd members to the
-cluster one by one.
+When using an external etcd host, you can restore the etcd backup by creating a
+new, single node etcd cluster. If using external etcd with multiple members, you
+can also add any additional etcd members to the cluster one by one. If you want
+to deploy etcd on master hosts, then there is no need create a new host. If you
+want to deploy dedicated etcd out of master hosts, then you need to create new
+hosts (external etcd).
```

Comment:
I'd actually leave this paragraph as it was before. The way it's been updated makes it sound as if you have to provision new hosts and that's not the case. You just have to restore one host from backup and then add the additional members to the cluster.